app/jobs/after_version_write_job.rb

```diff
@@ -0,0 +1,29 @@
+class AfterVersionWriteJob < ApplicationJob
+  queue_as :default
+
+  def perform(version:)
+    version.transaction do
+      rubygem = version.rubygem
+      version.rubygem.push_notifiable_owners.each do |notified_user|
+        Mailer.gem_pushed(owner, version.id, notified_user.id).deliver_later
```

Comment:
There's chance `owner` is `nil` here. Is that ok? Mail would fail, but in the background.

Reply:
fixed!

Reply:
Is it? :thinking: If I understand it well, owner still could be `nil`.

Reply:
Mailer now handles nil